src/elements/content-uploader/DroppableContent.js

```diff
@@ -31,18 +31,24 @@ const dropDefinition = {
     /**
      * Validates whether a file can be dropped or not.
      */
-    dropValidator: ({ allowedTypes }: { allowedTypes: Array<string> }, { types }: { types: Array<string> }) =>
-        Array.from(types).some(type => allowedTypes.indexOf(type) > -1),
+    dropValidator: (
+        { allowedTypes }: { allowedTypes: Array<string> },
+        { types }: { types: Array<string> | DOMStringList },
+    ) => {
+        if (types instanceof Array) {
+            return Array.from(types).some(type => allowedTypes.indexOf(type) > -1);
+        }
+
+        const allowedList = allowedTypes.filter(allowed => types.contains(allowed));
```

Comment:
Is this still needed?

Reply:
Yes because the types object is not an array for non-IE browsers and has no indexOf method. For IE11, we have to use DOMStringList and its contains method